app/services/audio_processing_service.py

```
@@ -64,62 +65,94 @@ def process_audio_task(
+        ) as e:
+            logger.error(
+                "Task %s failed for identifier %s. Error: %s",
+                task_type,
+                identifier,
+                str(e),
+            )
+            repository.update(
+                identifier=identifier,
+                update_data={"status": TaskStatus.failed, "error": str(e)},
+            )
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)